dj_cqrs/admin.py

```diff
@@ -0,0 +1,44 @@
+class CQRSAdminMasterSyncMixin:
+    """
+    Mixin that includes, on AdminModel, a custom action that allows syncronization of master's
+    model items from Django Admin page.
+    """
+
+    def get_actions(self, request):
+        """
+        Overrided method from AdminModel class, to include in actions list the sync method.
```

Comment:
Overridden method from AdminModel class; it is used to include the sync method in the actions list.